prisma/schema.prisma

```diff
@@ -1947,6 +1998,163 @@ model HomeBlock {
   clones    HomeBlock[]   @relation("Clones")
 }
 
+enum Currency {
+  USD
+  BUZZ
+}
+
+enum BountyType {
+  ModelCreation
+  LoraCreation
+  EmbedCreation
+  DataSetCreation
+  DataSetCaption
+  ImageCreation
+  VideoCreation
+  Other
+}
+
+enum BountyMode {
+  Individual
+  Split
+}
+
+enum BountyEntryMode {
+  Open
+  BenefactorsOnly
+}
+
+model Bounty {
+  id                      Int             @id @default(autoincrement())
+  userId                  Int?
```

Comment:
Would we  not want a index on userId? Same with other fields (e.g. startsAt). Based on how people might be filtering bounties and how we're sorting bounties, we could benefit from some indexes. We would probably want to review the actual queries that are used.